.github/workflows/gradle.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        java: [11, 17]
+        java: [17]
```

Comment:
```suggestion
        java: [17, 21]
```

Reply:
@jo-elimu I'll do this in separate changes since the main purpose of this PR is to resolve build error because of Jacoco plugin.
Anyway, some notes regarding this:
1/ Since Android Gradle build tool is currently 8.0.x, building with Java 21 will fail because it's unsupported yet with this version.
2/ Upgrading Android Gradle build tool to 8.5.x will probably cause build to fail because the new Jacoco plugin doesn't seem to support Gradle 8.5 yet.

So, for now. we should merge this PR and look for solution of this in another one.